sdks/java/core/build.gradle

```diff
@@ -90,4 +90,6 @@ dependencies {
   shadowTest library.java.avro_tests
   shadowTest library.java.zstd_jni
   testRuntimeOnly library.java.slf4j_jdk14
+  compile 'io.airlift:aircompressor:0.16'
+  compile 'com.facebook.presto.hadoop:hadoop-apache2:3.2.0-1'
```

Comment:
LZO itself should have no dependency on anything related to Hadoop, Presto, or Facebook.
1. Why do we need to include this?
2. If `aircompressor` really needs this, why does it need it?

Reply:
This is included because LzoCodec class that has been used to create Input&Output streams is using some classes of the org.apache.hadoop package, which is a part of com.facebook.presto.hadoop.
Since the aircompressor is designed to also support optional hadoop configurations, hadoop is coming into picture(in our case, hadoop config is null).